resources/common/plugins/macro/sort-entries/plugin.json

```diff
@@ -86,4 +86,4 @@
     "scriptFiles": [
         "sort-entries.js"
     ]
-}
+}
```

Comment:
Sorry for the detailed thing, but could you retain the empty line at the end of file?

Reply:
Sure, looks like the formatter removed it without me noticing